src/main/java/weirdaddons/WeirdAddonsSettings.java

```diff
@@ -265,8 +273,8 @@
     public static boolean allowUnauthenticatedPlayers = false;
 
     /*@Rule(
-            desc = "Placing a redstone on top of redstone ore will act as a zero tick generator",
-            extra = {"If a number is entered, it will be the delay in ticks between pulses","Enabling this will cause all redstone ore blocks to be transparent"},
+            desc = "tor",
```

Comment:
Hummmm yes, might wanna revert this first xDD

Reply:
Lol, don't even know how I manged that